sumdbaudit/README.md

```diff
@@ -4,19 +4,58 @@ The clone tool downloads all entries from the
 [Go SumDB](https://blog.golang.org/module-mirror-launch) into a local SQLite
 database, and verifies that the downloaded data matches the log commitment.
 
-This auditor provides an example for how log data can be verifiably cloned, and
+## Background
+This is a quick summary of https://blog.golang.org/module-mirror-launch but is not
+intended to replace this introduction.
+If you have no context on Go SumDB, read that intro first :-)
+
+Go SumDB is a Verifiable Log based on Trillian, which contains entries of the form:
+```
+github.com/google/trillian v1.3.11 h1:pPzJPkK06mvXId1LHEAJxIegGgHzzp/FUnycPYfoCMI=
+github.com/google/trillian v1.3.11/go.mod h1:0tPraVHrSDkA3BO6vKX67zgLXs6SsOAbHEivX+9mPgw=
+```
+Every module & version used in the Go ecosystem will have such an entry in this log,
+and the values are hashes which commit to the state of the repository and its `go.mod`
+file at that particular version.
+
+Clients can be assured that they have downloaded the same version of a module as
+everybody else provided all of the following are true:
+ * The hash of what they have downloaded matches an entry in the SumDB Log
+ * There is only one entry in the Log for the `module@version`
+ * Entries in the Log are immutable / the Log is append-only
+ * Everyone else sees the same Log
+
+## Features
+This auditor provides an example for how Log data can be verifiably cloned, and
 demonstrates how this can be used as a basis to verify its
 [Claims](https://github.com/google/trillian/blob/master/docs/claimantmodel/).
-In the Go SumDB case, the auditor needs to check that no module+version appears
-in the log twice, as this would provide a vector for different clients to be
-presented with different expected checksums. 
+The Claims checked by this clone & audit tool are:
+ * SumDB Checkpoints/STHs properly commit to all of the data in the Log
+ * Committed entries are never modified
+ * The Log is append-only
```

Comment:
These two are the same aren't they?
Maybe change to:
```
   * The log is append-only; committed entries are never modified
```
If you're after hammering this point home?

Reply:
Hah, this is actually how I wrote it first (well, with a slash instead of a semicolon but YOLO)